master/internal/api_trials_intg_test.go

```diff
@@ -1066,13 +1066,12 @@ func TestTrialLogs(t *testing.T) {
 
 	select {
 	case <-done:
-		break
```

Comment:
Why are we removing the `break`? Is the select not inside a loop?

Reply:
useless break inside a case clause, see https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#useless-break